src/components/TextField/TextField.tsx

```diff
@@ -353,6 +371,7 @@ class TextField extends React.PureComponent<CombinedProps, State> {
             {input}
             {suffixMarkup}
             {characterCountMarkup}
+            {clearButtonMarkup}
```

Comment:
Curious what this looks like (:tophat:) with a suffix and a character count. Do things get pretty crazy? How's the alignment?

Reply:
Its a bit cramped, but seems okay. I adjusted the margins on the clear button and character count so both match the margins on the suffix:

![](https://screenshot.click/2019-05-09_20-59-10-5jfze.png)